sources/lib/api/gcp/bigquery/_external_table.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+"""Implemenents External Table functionality."""
+
+import gcp
+import _api
+import _csv_options
+import _job
+
+
+class ExternalTable(object):
```

Comment:
We might as well call this FederatedTable so people can connect the dots from the BQ docs. Not really a fan of that name.

I wish the concept that BQ would have introduced is that of a DataSource, so it would generalize to both DataSets and Tables... because I might have external table(s) that are related.